qdrant-landing/themes/qdrant/static/css/blocks/_documentation.scss

```diff
@@ -39,6 +39,11 @@
     h6 {
       font-size: 1rem;
     }
+
+    // Correct spacing of unordered list in table cells
+    & table tr td ul {
+        margin: 0 !important;
+    }
```

Comment:
I've made this CSS change to compact the unordered list in the table. Without it it's rendered in an ugly way.

@trean It would be awesome if you could confirm whether this is fine.

Reply:
@timvisee Why do we need `!important` here? Seems to work without it.